registry/storage/driver/testsuites/testsuites.go

```diff
@@ -1135,12 +1135,19 @@ func randomFilename(length int64) string {
 	return string(b)
 }
 
-func randomContents(length int64) []byte {
-	b := make([]byte, length)
-	for i := range b {
-		b[i] = byte(rand.Intn(2 << 8))
+// randomBytes pre-allocates all of the memory sizes needed for the test. If
+// anything panics while accessing randomBytes, just make this number bigger.
+var randomBytes = make([]byte, 96<<20)
+
+func init() {
+	// increase the random bytes to the required maximum
+	for i := range randomBytes {
+		randomBytes[i] = byte(rand.Intn(2 << 8))
 	}
-	return b
+}
+
+func randomContents(length int64) []byte {
```

Comment:
It seems weird for this to take an `int64` if it's using the argument to slice an array. Changing the type to `int` would avoid a cast.